### PR TITLE
feat: add json logging

### LINKED ISSUE #9: Add JSON logging
I think we might need to switch to a different logger in order to do this?

### PR DESCRIPTION
**Pull request by im-adithya**
Fixes #9

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -75,20 +76,24 @@ func ToUrl(identifier string) (string, string, string, error) {
...
 func main() {
   c := &Config{}
+	logger := log.New()
+	logger.SetFormatter(&log.JSONFormatter{})
 
   // Load configruation from environment variables
   err := godotenv.Load(".env")
   if err != nil {
-    fmt.Println("Failed to load .env file")
+		logger.Fatalf("Failed to load .env file: %v", err)
```
Comment (kiwiidb) [inline main.go]: In production there is no .env file, this should be info level, certainly not Fatal

File: main.go [hunk 2]
```diff
@@ -114,6 +119,9 @@ func main() {
     ln := c.QueryParam("ln")
     lnurlpUrl, keysendUrl, nostrUrl, err := ToUrl(ln)
     if err != nil {
+			logger.WithFields(log.Fields{
+				"lightningAddress": ln,
```
Comment (kiwiidb) [inline main.go]: Please change all camelcase to snake_case, to be consistent with our datadog log format.

**Review by kiwiidb — CHANGES_REQUESTED** [review]
snake case + fix fatal log at start

File: main.go (hunk 2 above)
Comment (im-adithya) [reply main.go]: Done!

File: main.go (hunk 1 above)
Comment (im-adithya) [reply main.go]: Done!

File: main.go [hunk 3]
```diff
@@ -122,7 +130,10 @@ func main() {
 		go func() {
 			lnurlp, lnurlpResponse, err = GetJSON(GetJSONParams{url: lnurlpUrl, wg: &wg})
 			if err != nil {
-				e.Logger.Errorf("%v", err)
+				logger.WithFields(log.Fields{
+					"lightning_address": ln,
```
Comment (rolznz) [inline main.go:134]: The format of the keys is different to e.g. NWC. Does it matter?

Comment (kiwiidb) [reply main.go:134]: Probably doesn't matter that much

**Review by kiwiidb — APPROVED** [review]

**Pull request merged**